### DISCUSSION TITLE
"# noqa-next: XYZ" ?

### DISCUSSION THREAD

**Discussion by buhtz** [astral-sh/ruff#10902] [Q&A] [answered]
I know from pylint that I can disable the next occurrence of an error via `# pylint: disable-next=XYZ`.

I coulnd't found something like this for ruff. It often violates my 80-chars-line-length-limit when I set a `# noqa: XYZ`  at the end.

**Comment by MichaReiser** [comment, marked as answer]
I see the appeal to this. We want to rework our suppression comments and allow more flexible suppressions (e.g. for a block https://github.com/astral-sh/ruff/issues/3711) and unify them across our tools (the formatter, isort, and the linter all use different suppression comments). But it will probably be a while before we have the time to work on this.
